<ui version="4.0" >
 <class>CrossOptionWindow</class>
 <widget class="QDialog" name="CrossOptionWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Crossing schemes option</string>
  </property>
  <layout class="QVBoxLayout" name="crossOptionWindowVLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="crossingSchemesVLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="crossingSchemesGroupBox" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title" >
        <string>Crossing schemes</string>
       </property>
       <layout class="QVBoxLayout" name="crossingSchemesGroupBoxVLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="leftMargin" >
         <number>6</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>6</number>
        </property>
        <property name="bottomMargin" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="VLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QFrame" name="frame" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2" >
             <property name="spacing" >
              <number>0</number>
             </property>
             <property name="margin" >
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="nbCrossesHLayout" >
               <property name="spacing" >
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="nbCrossesLabel" >
                 <property name="minimumSize" >
                  <size>
                   <width>154</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>154</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>Number of crosses to be done</string>
                 </property>
                 <property name="text" >
                  <string>Number of crosses</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="nbCrossesLeftHSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>6</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QRadioButton" name="nbCrossesAllRadioButton" >
                 <property name="minimumSize" >
                  <size>
                   <width>80</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>80</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>Maximum number of crosses to be done</string>
                 </property>
                 <property name="text" >
                  <string>Maximum</string>
                 </property>
                 <property name="checked" >
                  <bool>true</bool>
                 </property>
                 <property name="autoExclusive" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="nbCrossesMaxHSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>6</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QRadioButton" name="nbCrossesRadioButton" >
                 <property name="minimumSize" >
                  <size>
                   <width>18</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>18</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string/>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                 <property name="autoExclusive" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="nbCrossesRightHSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>6</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QSpinBox" name="nbCrossesSpinBox" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>60</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>60</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="minimum" >
                  <number>2</number>
                 </property>
                 <property name="maximum" >
                  <number>99999</number>
                 </property>
                 <property name="value" >
                  <number>2</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::MinimumExpanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3" >
             <property name="spacing" >
              <number>0</number>
             </property>
             <property name="margin" >
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="indContribHLayout" >
               <property name="spacing" >
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="indContribLabel" >
                 <property name="minimumSize" >
                  <size>
                   <width>154</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>154</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>Contribution of each individual</string>
                 </property>
                 <property name="text" >
                  <string>Contribution of individuals</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="indContribLeftHSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>6</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QRadioButton" name="indContribAllRadioButton" >
                 <property name="minimumSize" >
                  <size>
                   <width>80</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>80</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>Unlimited contribution of each individual</string>
                 </property>
                 <property name="text" >
                  <string>Unlimited</string>
                 </property>
                 <property name="checked" >
                  <bool>true</bool>
                 </property>
                 <property name="autoExclusive" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="indContribUnlimitedHSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>6</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QRadioButton" name="indContribNbRadioButton" >
                 <property name="minimumSize" >
                  <size>
                   <width>18</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>18</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string/>
                 </property>
                 <property name="autoExclusive" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="indContribRightHSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>6</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QSpinBox" name="indContribSpinBox" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>60</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>60</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>Each individual can be crossed only "n" times</string>
                 </property>
                 <property name="minimum" >
                  <number>1</number>
                 </property>
                 <property name="maximum" >
                  <number>999</number>
                 </property>
                 <property name="value" >
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::MinimumExpanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4" >
             <property name="spacing" >
              <number>0</number>
             </property>
             <property name="margin" >
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="criterionHLayout" >
               <property name="spacing" >
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="criterionLabel" >
                 <property name="minimumSize" >
                  <size>
                   <width>50</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>50</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>Criterion</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="criterionHSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>10</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QComboBox" name="criterionComboBox" >
                 <property name="minimumSize" >
                  <size>
                   <width>130</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>130</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>Molecular Score: genetic value predicted (all QTL)
Weight: genetic value predicted (QTL are weighted)
Utility Criterion: expected mean and variance at the next generation (favor heterezygous)</string>
                 </property>
                 <property name="sizeAdjustPolicy" >
                  <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
                 </property>
                 <item>
                  <property name="text" >
                   <string>Molecular Score</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>Weight</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>Utility Criterion</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="criterionRightSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5" >
             <property name="spacing" >
              <number>0</number>
             </property>
             <property name="margin" >
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="methodHLayout" >
               <property name="spacing" >
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="methodLabel" >
                 <property name="minimumSize" >
                  <size>
                   <width>50</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>50</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>Method</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="methodLeftHSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>10</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QComboBox" name="crossingMethodsComboBox" >
                 <property name="minimumSize" >
                  <size>
                   <width>130</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>130</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>Complete: all possible crosses between individuals in the selcted list (half-diallel)
Better Half: avoid crosses between the last selected genotypes (optimization of selection intensity)</string>
                 </property>
                 <property name="currentIndex" >
                  <number>0</number>
                 </property>
                 <item>
                  <property name="text" >
                   <string>Complete</string>
                  </property>
                 </item>
                 <item>
                  <property name="text" >
                   <string>Better Half</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="methodRightHSpacer" >
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CrossOptionWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CrossOptionWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
